_data/identity.yml

```diff
@@ -158,6 +158,15 @@ websites:
           text: "2FA is only available on paid plans."
       doc: https://www.securesafe.com/en/faq/general-questions/how-can-i-activate-2-factor-authentication/
 
+    - name: The Identity Hub
+      url: https://www.theidentityhub.com
```

Comment:
When I go to `https://www.theidentityhub.com` I get redirected to `https://shop.theidentityhub.com/`. Is that just me or is indeed your homepage `https://shop.theidentityhub.com/`?

Reply:
The home page is https://www.theidentityhub.com, the redirect to https://shop.theidentityhub.com has been introduced to separate the commercial front-end from the main site. The documentation site and the admin site are in the https://www.theidentityhub.com domain (https://www.theidentityhub.com/hub/Documentation and https://www.theidentityhub.com/hub/Authenticate).

Reply:
Hello,

Any news on this? Do we need to make some changes for this to be approved?

Regards,
The Identity Hub

Reply:
Well, as the url value you have set redirects to the shop subdomain I'm afraid the site can't be listed. So you'll either have to change the url value to `https://shop.theidentityhub.com/` or stop the redirection.

Reply:
I changed the url to shop.theidentityhub.com and added a new commit.